lib/common/error_private.c

```diff
@@ -48,6 +48,7 @@ const char* ERR_getErrorString(ERR_enum code)
     case PREFIX(frameIndex_tooLarge): return "Frame index is too large";
     case PREFIX(seekableIO): return "An I/O error occurred when reading/seeking";
     case PREFIX(dstBuffer_wrong): return "Destination buffer is wrong";
+    case PREFIX(semaphore): return "Semaphore is corrupted";
```

Comment:
prefer `id_status` naming convention for error codes,
such as `semaphore_corruption` for example.